### PR TITLE
Improving the about dialog for licencing

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: web/react/components/about_build_modal.jsx [hunk 1]
```diff
@@ -21,29 +21,38 @@ export default class AboutBuildModal extends React.Component {
 
         let title = (
             <FormattedMessage
-                id='about.teamEdtion'
-                defaultMessage='Team Edition'
+                id='about.teamEdtiont0'
```
Comment (hmhealey) [inline web/react/components/about_build_modal.jsx]: Mind fixing teamEdtion by changing it to teamEdition? I don't want to start another RECIEVE thing again :smile:

**Comment by crspeller** [comment]
@hmhealey But `receieve` is a perfectly fine way to spell it :)  Spelling fixed. Ready for re-review.

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**